playbooks/deploy.yml

```diff
@@ -31,6 +31,8 @@
     run_refstack: "{{ refstack | default(false) | bool }}"
     run_tempest: "{{ tempest | default(false) | bool }}"
 
+    ceph_stack: "{{ ceph_stack | default('ceph-ansible') }}"
```

Comment:
We have to use a different variable name here. I will touch this first and rename everything in e.g. `_tempest`.

Reply:
Please rebase & use `_ceph_stack` inside the play.

Reply:
@berendt Please have a look.